tests/e2e/monitoring/test_when_one_of_the_prometheus_node_down.py

```diff
@@ -0,0 +1,182 @@
+import logging
+
+import pytest
+
+from time import sleep
+
+from ocs_ci.ocs import constants, ocp, defaults
+from ocs_ci.framework.testlib import tier4, E2ETest
+from ocs_ci.ocs.resources import pvc, pod
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+from tests.helpers import create_pvc, create_pod, create_unique_resource_name
+from ocs_ci.ocs.monitoring import (
+    collected_metrics_for_created_pvc,
+    get_kube_pod_spec_volumes_persistentvolumeclaims_info_metric
+)
+from ocs_ci.utility import aws
+from tests.helpers import wait_for_resource_state
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
```

Comment:
We are no longer going to define fixtures using class references. Please see https://github.com/red-hat-storage/ocs-ci/blob/master/docs/fixture_usage.md

Reply:
removed